vobject/change_tz.py

```diff
@@ -31,51 +29,45 @@ def change_tz(cal, new_timezone, default, utc_only=False, utc_tz=icalendar.utc):
         for node in (start, end):
             if node:
                 dt = node.value
-                if (isinstance(dt, datetime) and
-                        (not utc_only or dt.tzinfo == utc_tz)):
+                if isinstance(dt, datetime) and (not utc_only or dt.tzinfo == utc_tz):
                     if dt.tzinfo is None:
                         dt = dt.replace(tzinfo=default)
                     node.value = dt.astimezone(new_timezone)
 
 
+def show_timezones():
+    for tz_string in pytz.all_timezones:
+        print(tz_string)
+
+
+def convert_events(utc_only, args):
+    print(f"Converting {'only UTC' if utc_only else 'all'} events")
```

Comment:
f-strings won't work for Python 2.7.

Should we save this for 1.x and Python 3.7+ or change it to use `.format()` or just `%`, and merge it for 0.9?